core/tauri-runtime-wry/src/lib.rs

```diff
@@ -2702,6 +2705,32 @@ pub fn center_window(window: &Window, window_size: TaoPhysicalSize<u32>) -> Resu
   }
 }
 
+fn parse_proxy(url: &str) -> Result<ProxyConfig> {
+  let proxy = Url::parse(url).unwrap();
+
+  if proxy.scheme() == "http" && proxy.path() == "" {
+    let host = proxy.host().unwrap();
+    let port = proxy.port().unwrap();
```

Comment:
```suggestion
    let host = proxy.host().unwrap_or_default();
    let port = proxy.port().unwrap_or_default();
```

Reply:
The url has no `unwrap_or_default()` method.

Reply:
my bad, it should be

```rs
    let host = proxy.host().map(|h| h.to_string()).unwrap_or_default();
    let port = proxy.port().map(|p| p.to_string()).unwrap_or_default();
```